src/exchanges/binance-base.js

```diff
@@ -44,6 +44,7 @@ class BinanceBase extends BasicClient {
     restThrottleMs = 1000,
     l2updateSpeed = "",
     l2snapshotSpeed = "",
+    batchTickers,
```

Comment:
Default to `true`, then we can maintain backwards compatibility with current functionality.

Reply:
default to true 👍